rtengine/PF_correct_RT.cc

```diff
@@ -9,6 +9,9 @@
 // optimized: September 2013, Ingo Weyrich
 // further optimized: February 2018, Ingo Weyrich
 //
+//  Ingo Weyrich March 1028: The above comment 'Chromatic Aberration Auto-correction' sounds wrong
```

Comment:
`s/1028/2018/` ?

Reply:
Alternatively, add that to [Wikipedia](https://en.wikipedia.org/w/index.php?title=1028&action=edit), there isn't much information on that year...

Reply:
@Floessie @Beep6581 I was faster :)